server/routes/viewModels/taskLists/dmTasks.js

```diff
@@ -58,5 +82,9 @@ module.exports = (licenceStatus) => {
     return rejectedAddressTaskList(licenceStatus)
   }
 
+  if (showMandatoryAddressChecksNotCompletedWarning) {
+    return mandatoryCheckTaskList(licenceStatus)
+  }
```

Comment:
Added new mandatory check task to Decision maker flow and conditionally set to return mandatory check warning page route if `showMandatoryAddressChecksNotCompletedWarning` is `true`